src/unix/linux/x11/dl/libjpeg.h

```diff
@@ -246,6 +56,12 @@ static bool libjpeg_global_init(void)
 		bool r = true;
 
 		LIBJPEG_SO = MTY_SOLoad("libjpeg.so.8");
+		if (!LIBJPEG_SO)
```

Comment:
i would have probably factored out a function that takes an array of strings, and attempts to `MTY_SOLoad` each in turn, because we do it for N libraries M times each. and, it would make it easy for us to add logging etc. in a central spot, without having to paste the logging code multiple times.